protocol/v2/qbft/instance/prepare.go

```diff
@@ -102,7 +102,7 @@ func getRoundChangeJustification(state *specqbft.State, prepareMsgContainer *spe
 
 // validSignedPrepareForHeightRoundAndRoot known in dafny spec as validSignedPrepareForHeightRoundAndDigest
 // https://entethalliance.github.io/client-spec/qbft_spec.html#dfn-qbftspecification
-func validSignedPrepareForHeightRoundAndRootIgnoreSignature(
+func validSignedPrepareForHeightRoundAndRoot(
```

Comment:
can you please explain this change? `validSignedPrepareForHeightRoundAndRootIgnoreSignature` was taken from the spec https://github.com/ssvlabs/ssv-spec/blob/main/qbft/prepare.go#L82

Reply:
@y0she it was based on this comment: https://github.com/ssvlabs/ssv/pull/2078#discussion_r1990188268

lmk if we should update the specs, or if I should restore the original func name

Reply:
I think in general, In most cases, which should follow the specs. And wait for them to update even if something is "wrong" unless it's necessary to function. Also I can't see how is this part of the scope of this PR.